tests/virtualization/xen/dom_install.pm

```diff
@@ -0,0 +1,52 @@
+# Copyright (C) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Prepare the dom0 metrics environment
+# Maintainer: Pavel Dostál <pdostal@suse.cz>
+
+
+use base "x11test";
+use xen;
+use strict;
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = @_;
+    select_console 'x11';
+
+    x11_start_program('xterm');
+    send_key 'super-up';
+
+    assert_script_run "ssh root\@javor.qam.suse.cz 'zypper -n in vhostmd'";
```

Comment:
Maybe it's better to use a variable for the server name, like `QAM_XEN_SUPPORT_SERVER`, set in the test suite and do a `get_required_var('QAM_XEN_SUPPORT_SERVER');`

Also maybe `SUT_IP` can be used?

Reply:
Now I use `QAM_XEN_HYPERVISOR` variable. I still also need to rename virtuals and their DHCP / DNS records.